### PR TITLE
Added Will Schweitzer to directory

### PR DESCRIPTION
**Pull request by WilliamSchweitzer**

### CODE REVIEW

**Comment by domfarolino** [comment]
Are you interested in having a bio at all? (Though it is not mandatory I suppose)

**Review by domfarolino — APPROVED** [review]

**Comment by WilliamSchweitzer** [comment]
Oh, I’ll add a little about me tomorrow!

File: _data/_2020_members.yml [hunk 1]
```diff
@@ -28,3 +28,8 @@
   github: samwisebuze
   short_bio: I'm a fourth year CS Major. I currently work as a full-stack web developer for Learn21. I am the president of the UC Game Development Group and server as Webmaster for ACM-W.
   website: https://samuelbuzas.com
+- name: Will Schweitzer
+  image_url: https://avatars3.githubusercontent.com/u/21173745
+  github: williamschweitzer
+  short_bio: I'm a Computer Science major that enjoys making things with code. I am interested in learning to write clean, efficient code in a variety of languages. Primarily C# and Javascript.
```
Comment (domfarolino) [inline _data/_2020_members.yml]: I don't think "Computer Science" is a proper noun, so perhaps it should not be capitalized.

**Review by domfarolino — APPROVED** [review]
Cool! One more nit, but either way I can't merge this because I don't have authority on this repo anymore. /cc @vennemar

File: _data/_2020_members.yml (hunk 1 above)
Comment (WilliamSchweitzer) [reply _data/_2020_members.yml]: Nice catch. I'll change that now.

**Pull request merged**